src/main/java/frc/lib/io/DistanceIOLaserCAN.java

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.lib.io;
+
+import static edu.wpi.first.units.Units.*;
+
+import au.grapplerobotics.ConfigurationFailedException;
+import au.grapplerobotics.LaserCan;
+import au.grapplerobotics.interfaces.LaserCanInterface;
+import au.grapplerobotics.interfaces.LaserCanInterface.Measurement;
+import au.grapplerobotics.interfaces.LaserCanInterface.RangingMode;
+import au.grapplerobotics.interfaces.LaserCanInterface.RegionOfInterest;
+import au.grapplerobotics.interfaces.LaserCanInterface.TimingBudget;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.units.measure.Distance;
+import edu.wpi.first.wpilibj.Alert;
+import edu.wpi.first.wpilibj.Alert.AlertType;
+import frc.lib.util.CanDeviceId;
+import frc.robot.Constants;
+import org.littletonrobotics.junction.Logger;
+
+/** Add your docs here. */
+public class DistanceIOLaserCAN implements DistanceIO {
```

Comment:
I would change how these are implemented, DistanceIO should act as a distance reporting tool, and make an alternative that is a BeamBrakeIOLaserCAN which returns a boolean